go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	google.golang.org/grpc v1.25.1
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
 	gopkg.in/yaml.v2 v2.2.2
-	k8s.io/api v0.0.0-20190918195907-bd6ac527cfd2
+	k8s.io/api v0.0.0-20191003000013-35e20aa79eb8
```

Comment:
Can we do this in two steps please.

1. Update the k8s client and api types, see also #2155 
2. Then do this PR.

Reply:
I submitted #2214 & #2213 which address these two points.